dnf/yum/rpmtrans.py

```diff
@@ -402,13 +414,27 @@ def _script_start(self, key):
         # TODO: this doesn't fit into libdnf TransactionItem use cases
         action = dnf.transaction.PKG_SCRIPTLET
         if key is None and self._te_list == []:
-            pkg = 'None'
+            pkg_currenly_being_handled = 'None'
         else:
             transaction_list = self._extract_cbkey(key)
-            pkg = transaction_list[0].pkg
+            pkg_currenly_being_handled = transaction_list[0].pkg
         complete = self.complete_actions if self.total_actions != 0 and self.complete_actions != 0 \
             else 1
         total = self.total_actions if self.total_actions != 0 and self.complete_actions != 0 else 1
+
+        if hasattr(key, "name"):
+            pkg_which_owns_this_scriptlet_name = key.name
+        else:
+            pkg_which_owns_this_scriptlet_name = key
+
+        # In case rpm trigger runs scriptlet of some other package
+        if pkg_which_owns_this_scriptlet_name != pkg_currenly_being_handled.name:
+            # Show only the pkg name, because we don't have the full nevra
+            # and if we search for the pkg we could show misleding information
+            pkg = pkg_which_owns_this_scriptlet_name
```

Comment:
The name of "pkg" is confusing. It is string or pkg object or key (mostly string). What about `pkg_or_key`?

Reply:
In progress function it is called `package`. Therefore not much of help. `pkg_descriptor`?

Reply:
I updated it to `pkg_or_key`. I think its better because `pkg_descriptor` does not express well that it still could be a whole package.